meta-mentor-staging/recipes-kernel/perf/perf.bbappend

```diff
@@ -1 +1,3 @@
 DEPENDS += "flex-native bison-native"
+REMOVE_MAN_RDEPENDS = "${@bb.utils.contains('INCOMPATIBLE_LICENSE', 'GPLv3','man', '', d)}"
+RDEPENDS_${PN}-doc_remove_mel = "${REMOVE_MAN_RDEPENDS}"
```

Comment:
If the change is mel-specific, it belongs in meta-mel, if not, _mel is pointless. Pick one.